docs/source/kubernetes/minikube/step-zero-minikube.md

```diff
@@ -0,0 +1,46 @@
+(minikube-k8s)
+
+# Kubernetes on minikube (for development only)
+
+[minikube](minikube) is a implementation of Kubernetes as a local cluster primarily target to the development of applications.
```

Comment:
I figure "development only" excludes the excellent use of minikube for trial purposes.

```suggestion
# Kubernetes on minikube

[minikube](minikube) can setup a Kubernetes cluster on a single computer. Using minikube can be suitable trial for new Kubernetes users and for development purposes, but its not meant to be used for production purposes.
```